src/components/ButtonGroups/ButtonGroupBase/__test__/ButtonGroupBase.test.tsx

```diff
@@ -0,0 +1,22 @@
+import ButtonGroupBase, { ButtonGroupBaseProps } from "..";
+import ButtonBase from "../../../Buttons/ButtonBase";
+
+import { render, screen } from "@testing-library/react";
+
+describe("<ButtonGroupBase />", () => {
+    let props: ButtonGroupBaseProps;
+
+    it("should exist children with primary color", () => {

```

Comment:
You're not testing for primary color here but your test says so

Reply:
Okay, my bad.